pipelines/setup.yml

```diff
@@ -154,7 +154,7 @@ jobs:
               - |
                 cp release-ci-tfstate/release-ci.tfstate updated-release-ci-tfstate/release-ci.tfstate
                 terraform init paas-release-ci/terraform
-                terraform apply \
+                echo terraform apply \
```

Comment:
what is this echo here? was this left here by accident?

Reply:
Sorry, by accident. We need to fix this, there is no way to deploy these pipelines in the `deployer.<env>...` concourse due some permissions creating buckets. I think we should implement some way to bypass this, or enable it, or use the same bucket for everybody.

Reply:
fixed.